VMS/Pages/Organization-Profile/ActFavourite.razor

```diff
@@ -0,0 +1,94 @@
+﻿@inject IJSRuntime JSRuntinme;
+
+<div class="container">
+    <div class="row act">
+        <h1>Các hoạt động <span>được yêu thích</span></h1>
+        <div class="owl-carousel owl-theme activity__carousel">
+            <div class="act-container item">
+                <div class="img-container">
+                    <img src="./img/OrganizationProfile/ActFavourite1.png" alt="" class="act-img">
+                </div>
+                <div class="act-content">
+                    <a href="@Routes.ActivityInfo" class="content ps-3">
+                        <h3>Dạy học cho trẻ em mồ côi tại làng trẻ em SOS</h3>
```

Comment:
Please don't hard code those values like this anymore, please bind data using foreach loop.